crates/type-system/src/ontology/entity_type/mod.rs

```diff
@@ -75,6 +78,11 @@ impl EntityType {
         self.description.as_deref()
     }
 
+    #[must_use]
+    pub const fn inherits_from(&self) -> &AllOf<EntityTypeReference> {
```

Comment:
Do we want to return a slice instead?

Reply:
I wondered that as well, but I think there's _some_ value in having the `AllOf` rather than `&[EntityTypeReference]` because we may add more information/implementations to `AllOf`. When we look at inheritance we're going to want some logic about composing the types involved. 

_**Probably**_ it won't happen in the `AllOf` object, but it's hard to tell, so might as well keep the object. I had previously actually just had `inherits_from: Vec<EntityTypeReference>` but we're doing something else for `OneOf` so I'd prefer to be consistent for now.

Reply:
fair 🙂